bubble.py

```diff
@@ -0,0 +1,13 @@
+def bubbleSort(numbers):
```

Comment:
@ankitjena - The program already exists https://github.com/hacktoberfest17/programming/blob/master/sorting/bubble_sort/bubble_sort.py